en/docs/quick-starts/saml-front-channel-logout.md

```diff
@@ -101,14 +95,14 @@ The user is required to log in to two different applications created in a tenant
 
     ![choose-localhost](../../assets/img/samples/choose-localhost.png)
 
-3.  Login as tenant admin. You will be directed to the page with authenticated user information. 
+3.  Login as admin. You will be directed to the page with authenticated user information. 
```

Comment:
Login --> Log in

"login" when its used as a noun for e.g., "Use your login credentials" and "log in" when used as a verb for e.g., "Log in to the application"